### PR TITLE
Implemented swiper.

### PR DESCRIPTION
**Pull request by ocampd1**

### CODE REVIEW

File: chucks-pick3-mobile/src/app/pages/home/home.page.html [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
-        </ion-slide>
-        <ion-slide style="background-color: white;">
-            <ion-router-outlet name="card-page-6"></ion-router-outlet>
-        </ion-slide>
-        <ion-slide style="background-color: white;">
-            <ion-router-outlet name="card-page-7"></ion-router-outlet>
-        </ion-slide>
-    </ion-slides>
+
+  <swiper #pick3DrawDateCards [config]="slideOpts" [slidesPerView]="1" style="height: 100%" (slideChange)="initializePick3DrawDateCard($event)"
```
Comment (pouncilt) [inline chucks-pick3-mobile/src/app/pages/home/home.page.html:3]: Should we use Style class instead of in-line style here?

**Review by pouncilt — CHANGES_REQUESTED** [review]
Also ensure the code coverage is up to par.

File: chucks-pick3-mobile/src/app/pages/home/home.page.ts [hunk 2]
```diff
@@ -11,125 +11,146 @@ import {DrawDateService} from '../../services/draw-date.service';
...
+        /* console.log(`HomePage.initializePick3DrawDateCard() method activeIndex:${activeIndex}, (${activeIndex} + 1) = ${activeIndex + 1}`);*/
+
+        if (activeIndex <= 6) {
+          this.cardContextService.addContext({
+            slideNumber: activeIndex + 1,
+            data: pick3DrawDateDecks[activeIndex],
+            defaultDrawDateTime: this.default.drawDateTime,
+            drawTimes: this.appService.getPick3DrawTimeCards(activeIndex + 1)
+          });
+        } else {
```
[the remark this answers is not in the document]
Comment (ocampd1) [reply chucks-pick3-mobile/src/app/pages/home/home.page.ts]: Removed

File: chucks-pick3-mobile/src/app/pages/home/home.page.ts [hunk 3]
```diff
@@ -11,125 +11,146 @@ import {DrawDateService} from '../../services/draw-date.service';
...
+  public initializePick3DrawDateCard(swiper): void {
+    let activeIndex;
+    activeIndex = swiper.activeIndex;
+    if (this.slidesLoaded) {
+      this.storeId();
+        console.log(swiper.activeIndex);
+        activeIndex = swiper.activeIndex;
+        // TODO Determine if we are swiping left(activeIndex -1) or right (activeIndex +1) on slides
+        console.log(`HomePage.initializePick3DrawDateCard() - Active Index: IonSlides[${activeIndex}]`);
+        // const nextPick3DrawDate = this.pick3DrawDateDecks[activeIndex];
```
[the remark this answers is not in the document]
Comment (ocampd1) [reply chucks-pick3-mobile/src/app/pages/home/home.page.ts]: Removed

File: chucks-pick3-mobile/src/app/pages/home/home.page.ts [hunk 4]
```diff
@@ -11,125 +11,146 @@ import {DrawDateService} from '../../services/draw-date.service';
...
-                }
-            });
+  public initializePick3DrawDateCard(swiper): void {
+    let activeIndex;
+    activeIndex = swiper.activeIndex;
+    if (this.slidesLoaded) {
+      this.storeId();
+        console.log(swiper.activeIndex);
+        activeIndex = swiper.activeIndex;
+        // TODO Determine if we are swiping left(activeIndex -1) or right (activeIndex +1) on slides
```
[the remark this answers is not in the document]
Comment (ocampd1) [reply chucks-pick3-mobile/src/app/pages/home/home.page.ts]: No longer needed, removed

File: chucks-pick3-mobile/src/app/pages/home/home.page.ts [hunk 5]
```diff
@@ -11,125 +11,146 @@ import {DrawDateService} from '../../services/draw-date.service';
...
-    ngAfterViewInit() {
-        const routerState = this.router.getCurrentNavigation().extras.state;
-        if (routerState && this.router.url === '/home') {
-            const currentSlideNumber = this.appService.pick3CardId;
-            if (currentSlideNumber) {
-                this.next(currentSlideNumber - 1);
-            }
-        }
-    }
+  ngAfterViewInit() {
```
[the remark this answers is not in the document]
Comment (ocampd1) [reply chucks-pick3-mobile/src/app/pages/home/home.page.ts]: Removed method

File: chucks-pick3-mobile/src/app/pages/home/home.page.ts [hunk 6]
```diff
@@ -11,125 +11,146 @@ import {DrawDateService} from '../../services/draw-date.service';
...
+    speed: 400,
+    slidesPerView: 1,
+    initialSlide: 7
+  });
+
+
+  default = {
+    drawDateTime: Pick3DrawTimeEnum.MORNING
+  };
+  slideOpts = {
```
[the remark this answers is not in the document]
Comment (ocampd1) [reply chucks-pick3-mobile/src/app/pages/home/home.page.ts]: Removed from code

**Pull request merged**